### DISCUSSION TITLE
don't show the date when i change the value

### DISCUSSION THREAD

**Discussion by rediff666** [henninghall/react-native-date-picker#496] [General]
**Describe the bug**
I can't get the new value of the date in a Text Object or Alert message

**To Reproduce**
const [date, setDate] = useState(new Date())
const [open, setOpen] = useState(false)

return (
    <>
       <Text> Date Value : {date.toString()}</Text>
      <Button title="Open" onPress={() => setOpen(true)} />
      <DatePicker
        modal
        open={open}
        date={date}
        onConfirm={(date) => {
          setOpen(false)
          setDate(date)
        }}
        onCancel={() => {
          setOpen(false)
        }}
      />
    </>
  )
}
```

**Smartphone (please complete the following information):**
 - OS: [Android or iOS]
 - React Native version [e.g. 0.59.4]
 - react-native-date-picker version [e.g. 2.5.1]

**Comment by henninghall** [comment]
It's a little bit hard to understand what issue you're having here. I suggest you describe more in detail and send a screenshot/screencap.

**Reply by Masshen** [reply]
I'm having the same problem: onConfirm, instead of returning a date, returns an empty object {}.
Here are the app specifications:
- react-native@0.78.0
- react-native-date-picker@5.0.12
